### PR TITLE
Update reference to bounds.

### PR DESCRIPTION
**Pull request by daspecster**
Towards #2753 

Old references to `bounding_poly` -> `bounds`.

### CODE REVIEW

**Comment by daspecster** [comment]
Cancelled CI since it's just a docs change.

File: docs/vision-usage.rst [hunk 1]
```diff
@@ -169,9 +169,9 @@ will also provide their known geographical locations if available.
     -33.857123
     >>> landmarks[0].locations[0].longitude
     151.213921
-    >>> landmarks[0].bounding_poly.vertices[0].x_coordinate
+    >>> landmarks[0].bounds.vertices[0].x_coordinate
     78
-    >>> landmarks[0].bounding_poly.vertices[0].y_coordinate
+    >>> landmarks[0].bounds.vertices[0].y_coordinate
```
Comment (tseaver) [inline docs/vision-usage.rst]: I was confused here that these were instances of `face.Landmark`, which have no `bounds` attribute:  but these are `entity.EntityAnnotation` instances, right?  Is it just that the API reuses the term confusingly, and we follow suit?

Comment (daspecster) [inline docs/vision-usage.rst]: `Landmark` actually uses [`Position`](https://github.com/GoogleCloudPlatform/google-cloud-python/blob/5d6148e645a38691eedcd78ae20fd011aa001bb6/vision/google/cloud/vision/geometry.py#L109) which is a 3D point.  [`Bounds`](https://github.com/GoogleCloudPlatform/google-cloud-python/blob/5d6148e645a38691eedcd78ae20fd011aa001bb6/vision/google/cloud/vision/geometry.py#L18) is actually a container for a list of points which make a path that encompasses the boundaries of the detected object or feature.

It is complex, and if you have ideas to make it simpler I can make an issue and I'm more than happy to implement ideas!

Comment (daspecster) [inline docs/vision-usage.rst]: @tseaver I updated #2753 to mention this.

**Comment by daspecster** [comment]
Yeah, I'm trying to clean those up as I go.

**Pull request merged**